parts/footer-alpha-v3.html

```diff
@@ -12,71 +14,75 @@
-	<!-- /wp:paragraph -->
-	
-	<!-- wp:paragraph {"style":{"elements":{"link":{"color":{"text":"var:preset|color|text-color"},":hover":{"color":{"text":"var:preset|color|tertiary"}}}},"spacing":{"margin":{"top":"1rem","bottom":"0.15rem"}},"typography":{"fontSize":"0.85rem"}},"textColor":"text-color"} -->
-	<p class="has-text-color-color has-text-color has-link-color" style="margin-top:1rem;margin-bottom:0.15rem;font-size:0.85rem"><a href="https://www2.gov.bc.ca/gov/content/home/get-help-with-government-services">Contact us</a></p>
-	<!-- /wp:paragraph --></div>
-	<!-- /wp:column --></div>
-	<!-- /wp:columns -->
-	
-	<!-- wp:separator {"style":{"spacing":{"margin":{"top":"2rem","bottom":"2rem"}}},"backgroundColor":"septenary","className":"is-style-wide"} -->
-	<hr class="wp-block-separator has-text-color has-septenary-color has-alpha-channel-opacity has-septenary-background-color has-background is-style-wide" style="margin-top:2rem;margin-bottom:2rem"/>
-	<!-- /wp:separator -->
-	
-	<!-- wp:group {"style":{"spacing":{"padding":{"top":"0"},"margin":{"top":"0"}}},"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"space-between","verticalAlignment":"top"}} -->
-	<div class="wp-block-group" style="margin-top:0;padding-top:0"><!-- wp:group {"layout":{"type":"flex","flexWrap":"nowrap"}} -->
-	<div class="wp-block-group"><!-- wp:social-links {"iconColor":"nonary","iconColorValue":"var(\u002d\u002dbcds-icons-icons-primary)","iconBackgroundColor":"background","iconBackgroundColorValue":"var(\u002d\u002dbcds-surface-background-white)"} -->
-	<ul class="wp-block-social-links has-icon-color has-icon-background-color"><!-- wp:social-link {"url":"https://www.facebook.com/BCProvincialGovernment/","service":"facebook","className":"icon"} /-->
-	
-	<!-- wp:social-link {"url":"https://www.instagram.com/governmentofbc","service":"instagram","className":"icon"} /-->
-	
-	<!-- wp:social-link {"url":"https://twitter.com/BCGovNews","service":"x","className":"icon"} /--></ul>
-	<!-- /wp:social-links --></div>
-	<!-- /wp:group -->
-	
-	<!-- wp:paragraph {"align":"left","style":{"spacing":{"padding":{"top":"0.5rem","left":"0rem"}},"typography":{"fontSize":"1rem"}},"textColor":"text-color"} -->
-	<p class="has-text-align-left has-text-color-color has-text-color" style="padding-top:0.5rem;padding-left:0rem;font-size:1rem">© 2023 Government of British Columbia.</p>
+
+<!-- wp:column {"width":"15%"} -->
+<div class="wp-block-column" style="flex-basis:15%"><!-- wp:heading {"style":{"spacing":{"margin":{"top":"0","right":"0","bottom":"0","left":"0"},"padding":{"bottom":"var:preset|spacing|10"}},"elements":{"link":{"color":{"text":"var:preset|color|text-color"}}},"typography":{"fontStyle":"normal","fontWeight":"600"}},"textColor":"text-color","className":" text-uppercase","fontSize":"extra-small"} -->
```

Comment:
why did you change the font-size from rem's to normal, or did wordpress do this automatically

Reply:
wordpress did that thru the gutenberg editor